src/app.ts

```diff
@@ -599,9 +650,75 @@ function fetchDescriptionGenerationData(
       ),
     }).predict(prompt)
   );
+  const descriptionKeywordPrefix =
```

Comment:
Shouldn't this only be done if the "validate" checkbox was on?

Reply:
now, THAT's for few-shot exampling of descriptions (optional)